packages/frontend-2/components/viewer/saved-views/panel/views/Group.vue

```diff
@@ -140,12 +141,17 @@ const { triggerNotification } = useGlobalToast()
 const isLoading = useMutationLoading()
 const createView = useCreateSavedView()
 const updateGroup = useUpdateSavedViewGroup()
+const updateView = useUpdateSavedView()
 const renameMode = defineModel<boolean>('renameMode')
 
 const open = defineModel<boolean>('open')
 const showMenu = ref(false)
 const menuId = useId()
 
+// Drag and drop state
+const isDragOver = ref(false)
```

Comment:
[would be nicer](https://vuejs.org/guide/reusability/composables#extracting-composables-for-code-organization) to keep all of this in its own composable under `~/lib` so that dragging internals aren't mixed together with all of the other logic